superbuild/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
I'm not against it but maybe it's worth making sure the glibc version is old enough?

Reply:
It is as old as 20.04 was old when we picked it. We will see if anyone complains.